hubbot

```diff
@@ -417,15 +417,17 @@ def expire_work_dirs(github_visible_commits):
 
 def expire_images():
     images_dir = os.path.join(test_data, "images")
-    expired_dir = os.path.join(test_data, "images.expired")
-    if not os.path.exists(images_dir):
-        return
+    mock_config_dir = os.path.join(test_data, "mock")
 
-    keep = { }
+    kept_image_files = { }
+    kept_mock_configs = { }
 
     def keep_image(flavor, os_, tag):
-        keep["%s-%s-*-%s-checksum" % (flavor, os_, tag)] = True
-        keep["%s-%s-*-%s.*" % (flavor, os_, tag)] = True
+        kept_image_files["%s-%s-*-%s-checksum" % (flavor, os_, tag)] = True
+        kept_image_files["%s-%s-*-%s.*" % (flavor, os_, tag)] = True
```

Comment:
(Hmm, tags currently are just strings, without any ordering etc.)  Could we do this in a separate PR?